packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -1164,6 +1164,17 @@ export class ChannelCollection implements IFluidDataStoreChannel, IDisposable {
 		}
 	}
 
+	public deleteChild(dataStoreId: string) {
```

Comment:
Who will call this?

Reply:
GC?

Reply:
I'd personally separate GC from managing nodes. GC tells this object what paths/routes should be cleared, it's up to this object to map that to nodes and how to delete them.
The answer to your question would be - whenever this object wants / needs to delete nodes.

In my case that would be when temporal collab space needs to be destroyed. There would be no handle / GC business for such spaces, and thus I'll control fully lifetime of them (they are short lived).

Reply:
Sounds good. I wanted to understand who will call this because it's public and no usages are added in this PR.